app-emacs/org-roam/org-roam-2.2.2.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+NEED_EMACS=26.1
+inherit elisp
+
+DESCRIPTION="Rudimentary Roam replica with Org-mode"
+HOMEPAGE="https://github.com/org-roam/org-roam"
+SRC_URI="
+	https://github.com/org-roam/org-roam/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	app-emacs/dash
+	app-emacs/magit
+	app-emacs/emacsql[sqlite]
+"
+DEPEND="${RDEPEND}"
```

Comment:
Should be `BDEPEND` here, because byte-compilation takes place using Emacs on the build host.